aten/src/TH/generic/THBlas.cpp

```diff
@@ -380,6 +383,39 @@ void THBlas_(gemm)(
     return;
   }
 #endif
+
+#ifdef USE_FBGEMM
+#if defined(TH_REAL_IS_LONG)
+  if (alpha == 1 && (beta == 0 || beta == 1)) {
+    bool accumulate;
```

Comment:
I think this can just be `bool accumulate = beta != 0;` or even just in-place could use `!beta`

Reply:
Updated. Thanks!